Sources/MapboxNavigation/MultiplexedSpeechSynthesizer.swift

```diff
@@ -40,6 +40,18 @@ open class MultiplexedSpeechSynthesizer: SpeechSynthesizing {
         speechSynthesizers.forEach { $0.locale = locale }
     }
     
+    /// Controls if this speech synthesizer is allowed to manage the shared `AVAudioSession`.
+    /// Set this field to `false` if you want to manage the session yourself, for example if your app has background music.
+    /// Default value is`true`.
+    public var managesAudioSession: Bool {
+        get {
+            speechSynthesizers.reduce(true, { $0 && $1.managesAudioSession })
```

Comment:
NIT: What do you think about such approach?
```suggestion
            !speechSynthesizers.contains(where: { !$0.managesAudioSession })
```

Reply:
Agree, might be easier to understand